.github/workflows/test_release.yml

```diff
@@ -71,19 +57,24 @@ jobs:
       - name : Push Docker Image
         run: docker push ${{ env.ACR_REPO }}/${{ env.PREFIX }}:latest
 
-      - name: Restart Azure Functions App
-        run: az functionapp restart --resource-group ${{ env.RESOURCE_GROUP }} --name ${{ env.PREFIX }}-functionapp
-
       - name: Set SFTP Environment Variables
         run: |
               az functionapp config appsettings set \
               --resource-group ${{ env.RESOURCE_GROUP }} \
               --name ${{ env.PREFIX }}-functionapp \
-              --settings PRIME_ENVIRONMENT=${{ env.PRIME_ENVIRONMENT }} ${{ env.SFTP_CREDS }}
+              --settings PRIME_ENVIRONMENT=${{ env.PRIME_ENVIRONMENT }} ${{ env.SFTP_CREDS }} \
+              --output none \
+              --only-show-errors
 
       - name: Set Redox Environment Variables
         run: |
               az functionapp config appsettings set \
               --resource-group ${{ env.RESOURCE_GROUP }} \
               --name ${{ env.PREFIX }}-functionapp \
-              --settings REDOX_SECRET=${{ env.REDOX_SECRET }} 
+              --settings REDOX_SECRET=${{ env.REDOX_SECRET }} \
+              --output none \
+              --only-show-errors
+
+      - name: Restart Azure Functions App
```

Comment:
I moved the restart after setting env vars.  In rare cases, the env vars may not set because the restart rendered the function app unreachable.